mlflow/langchain/api_request_parallel_processor.py

```diff
@@ -434,3 +522,38 @@ def process_api_requests(
             )
 
         return [res for _, res in sorted(results)]
+
+
+def process_stream_request(
+    lc_model,
+    request_json: Union[Any, Dict[str, Any]],
+    callback_handlers: Optional[List[BaseCallbackHandler]] = None,
+    convert_chat_responses: bool = False,
+):
+    from mlflow.langchain.utils import lc_runnables_types
+
+    if not isinstance(lc_model, lc_runnables_types()):
+        raise MlflowException(
+            f"Model {lc_model.__class__.__name__} does not support streaming prediction output."
+        )
+
+    results: list[tuple[int, str]] = []
+    errors: dict = {}
```

Comment:
why do we need type hints on these variables?

Reply:
no need. I will remove them